### ISSUE TITLE
The year of the about box is hardcoded to 2016

### ISSUE THREAD

**Issue by DazzyWalkman** [BasicAirData/GPSLogger#159]
ATT. Release 3.1.0.
This does not affect any functionality, just looks   a little odd. 
Maybe we can introduce sort of a variable denoting the year of the release in the string 'about_description'？

**Comment by GrazianoCapelli** [comment]
A valid copyright notice contains three elements:

1) The copyright symbol ©, or the words "Copyright" or "Copr."
2) The year of first publication
3) The name of the copyright owner

It is not required that these elements appear in any particular order in the notice, but most notices are written in the order set forth above. The topic is well explained here: https://www.copyrightlaws.com/copyright-symbol-notice-year/

Luckily no variable is needed: one less job for programmers and translators!

**Comment by zedxxx** [comment]
How about to use a range of years (i.e. "2016-2022")?

https://www.gnu.org/licenses/gpl-howto.en.html

> For software with several releases over multiple years, it's okay to use a range (“2008-2010”) instead of listing individual years (“2008, 2009, 2010”) if and only if every year in the range, inclusive, really is a “copyrightable” year that would be listed individually; and you make an explicit statement in your documentation about this usage.

**Comment by GrazianoCapelli** [comment]
We would need to do some changes to about screen:
1) We could use: _Copyright © 2016-2022 BasicAirData_: the year range seems a good idea, as the GPL suggests.
2) We should change the http links to https
3) Remove the reference to "/GPSLogger" folder in favour of a more generic text, because now the local exportation folder is selectable (in Android 5+).

- Furthermore, maybe we should put a small note on translation, with the link of the Crowdin page.

**Comment by GrazianoCapelli** [comment]
The changes have been merged to the develop branch, and will be included into the next v3.1.3.
Thanks @DazzyWalkman and @zedxxx for your useful suggestions!

**Comment by GrazianoCapelli** [comment]
@DazzyWalkman - in my first answer it seems that I know everything about copyright notice, so I want to explain and, in case, apologize for the wrong approach I took.
Actually I consider the Copyright (and all about laws and regulations) a very sensitive, unfriendly and unknown topic for me, and for this reason I spent some time to try to inform myself before writing the About Screen.
Writing the About description, I sweated more on laws than on coding.
To date I'm not sure that I written all the right way, and for this reason I tend first of all to justify my choice and stay stick to what it seems more correct (or, at least, less wrong) for me.

Anyway, I apologize for it and I thank you again for your cooperation and your reporting.
